cumulus_library/study_parser.py

```diff
@@ -134,21 +147,25 @@ def get_export_table_list(self) -> Optional[StrList]:
                 )
         return export_table_list
 
-    def reset_export_dir(self) -> None:
```

Comment:
`*_export_dir` has been dangling for a bit unused - fixed the namespace and made it actually referenced.